app/services/ping.rb

```diff
@@ -47,6 +50,39 @@ def ping_database
       }
     end
 
+    def ping_cache
+      response = {}
+
+      if redis_cache_store?
+        redis = Rails.cache.redis
+        response['servers'] = {}
```

Comment:
This results in an Hash with key index and value "the data". is that intentional?

Reply:
When I change this to `[]`, I can make hammer output the fields fine with:

```
          field :cache, _('cache'), Fields::Label do
            field :servers, _('servers'), Fields::Label do
              field :status, _('Status')
              field :duration_ms, _('Server Response')
            end
          end
```

Reply:
but turning this into an array wouldn't leak anything?
you'd end up with an output like
```
cache:    
    servers: 
     1) Status:          ok
        Server Response: 0
     2) Status:          fubar
        Server Response: 0

Reply:
Fair.